ecosystem/indexer-grpc/indexer-grpc-file-store/README.md

```diff
@@ -20,13 +20,13 @@ File store fetches data from cache and stores in Google Cloud Storage.
 * A Redis cache running at `$REDIS_ADDRESS`, e.g., `127.0.0.1:6379`
 * Example command to run:
 
-```bash
-fullnode_grpc_address: 127.0.0.1:50051
-redis_address: 127.0.0.1:6379
-health_check_port: 8083
-file_store:
-  file_store_type: GcsFileStore
-  gcs_file_store_bucket_name: indexer-grpc-file-store-testnet 
+```yaml
+health_check_port: 8082
+server_config:
```

Comment:
maybe also update the local filestore docs below too? config is out of date and also it doesn't necessarily have to be "test only" now if folks want to run it with a local ssd ;)

Reply:
good catch; updated